### ISSUE TITLE
Would be nice to see the calling order on the visualisation

### ISSUE THREAD

**Issue by offworldcoder** [findtheflow/Feedback#17]
This could be indicated by numbers similar to a communication diagram (http://www.uml-diagrams.org/communication-diagrams.html)

**Comment by YoannBuch** [comment]
You're right, there is currently no ways to know when calls are being made between the different nodes (package/class/method) of the call graph. 

If you want to follow the order of calls, you can use the call stack right below the call graph. Selecting the calls will highlight the corresponding package/class/method in the call graph.That being said I can understand that it may not be intuitive and too detailed.

Numbering calls like in communication diagrams is something to consider. I feel like it works well on small graphs but it would probably be really hard to follow on the kinds of call graphs we have in Flow. I'm not sure about your case, but I often end up with graphs with dozens of nodes and relationships. And one relationship that you see on the screen may actually represent hundreds of calls, and sometimes made at different times. I'm not sure that that numbering scheme would support the messy reality that Flow is uncovering. I have to think more about it, especially about how to handle iterations.

How many numbers would you have on the typical graph you generate with Flow?

Maybe we could implement something a little bit more interactive to follow the flow of execution on the call graph. For example we could have some sort of a handle on the timeline that when grabbed could slide along the time, and while doing so would highlight the node in the call graph that is being executed at the given time. What do you think? I'm open to other suggestions as well :)
